### PR TITLE
Shorter SCSP behaviour review duration

### PR DESCRIPTION
**Pull request by GordonF42**
Since we are early in development and there will be a lot of SCS for adding new features, which may also depend on other SCS. The current 2-week review duration will probably slow down a bit too much the initial development process.
The review duration will probably be restored to 2 weeks when the ground work is finished.

Note: Specs bot will need to be updated to display 7 days instead of 14 when this is merged.

### CODE REVIEW

File: content/introduction/scsp.md
Comment (vabd) [inline content/introduction/scsp.md]:
```suggestion
total of 7 days (one week). During this period, anyone **can** issue a
```

**Review by vabd — CHANGES_REQUESTED** [review]
On top of that, please expand each line to as close to 80 char as possible (while keeping it under or equal to this length) to keep the formatting consistent trough the whole documentation.

FTR this would turn the end of the file into:

```
total of 7 days (one week). During this period, anyone **can** issue a review on
the SCS or comment and discuss it, preferably in the pull request's comments.
```

**Comment by GordonF42** [comment]
my bad, should be good now

**Review by vabd — APPROVED** [review]
lgtm, let's wait till the end of the 14 day-long call for review before merging it

**Comment by vabd** [comment]
Merging today. Might be worth noting that this new rule only applies for SCSs that have been open after the merging happens (the ones that have been open before that are still subject to the full 14 days rule).

**Pull request merged**
